Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -279,6 +265,11 @@ - (void)setupToolbar {
 }
 
 - (void)updateToolbarItemsIfNeeded {
+    
+    if (!self.saveButtonController) {
```

Comment:
sure, i would argue there is an impact but will concede that the main thing we're missing here is a (UI) test to verify save buttons are hooked up properly and are functional.